style.css

```diff
@@ -304,51 +313,79 @@ display: none;
   }
 
   .swap-class {
-    order: 4;
+    order: 1;
   }
-h1 {
- justify-content: end;
- 
-}
- 
-p {
-  width: 60%;
 
- }
+  #fs-frm {
+    max-width: 45%;
+    margin: 0 auto;
+  }
 
- .connect {
-  width: 60%;
-  color: #7f8cff;
-  margin-bottom: 0;
-  text-decoration: none;
-}
+  .header-img {
+    text-align: center;
+    border-top-left-radius: 120px;
+    padding-top: 2%;
+    background-position: center;
+  }
```

Comment:
- [ ] Please 🙏🏽 complete the required changes from the last code reviewer, your design must look like the Figma design:

> Hey, please, let's make some adjustments by adding the correct background image and reducing the width of the text so as for the text not to be too much apart from the inputs. I made a highlight in the images below so you can see how to adjust it. Let's fix this to improve our design as much as we can below:
> Let's remove the black border on the input to improve our design to match the Figma design as much as we can.
> Let's center the button to improve our design below:

| Figma design | Your design |
|---|---|
| ![image](https://user-images.githubusercontent.com/58642949/208251140-16155fa5-b175-41c6-9c80-e520f8252f12.png) | ![image](https://user-images.githubusercontent.com/58642949/208251356-009f6a4f-7fc3-4008-ab98-9b6ab1f45470.png) |